cmd/vcsim/main.go

```diff
@@ -35,12 +37,15 @@ func main() {
 	flag.IntVar(&model.Pool, "pool", model.Pool, "Number of resource pools per compute resource")
 
 	isESX := flag.Bool("esx", false, "Simulate standalone ESX")
+	isTLS := flag.Bool("tls", false, "Enable TLS")
```

Comment:
I see we are heavily using builtin flag module for some reason. Is there really any need for that? I always feel built-in golang flag module is a little clunky. I think go-flags package is much nicer.